res/controllers/Hercules-DJ-Console-RMX-scripts.js

```diff
@@ -450,6 +450,19 @@ HerculesRMX.Deck.prototype.cueSelectHandler = function(value) {
    }
 };
 
+HerculesRMX.Deck.prototype.sourceSelectHandler = function(value) {
```

Comment:
is there a reason this has to be a script binding and not just a basic toggle button?